modules/internal/DefaultRectangular.chpl

```diff
@@ -1036,13 +1036,24 @@ module DefaultRectangular {
     }
 
     proc dsiReallocate(d: domain) {
+      on this {
+      //
+      // If both d and dom are default rectangular, this is pretty
+      // easy...
+      //
       if (d._value.type == dom.type) {
-        on this {
         var copy = new DefaultRectangularArr(eltType=eltType, rank=rank,
-                                            idxType=idxType,
-                                            stridable=d._value.stridable,
-                                            dom=d._value);
-        for i in d((...dom.ranges)) do
+                                             idxType=idxType,
+                                             stridable=d._value.stridable,
+                                             dom=d._value);
+        //
+        // TODO: Making this for into a forall ought to accelerate
+        // dsiReallocate() calls, yet doing so breaks due to uint/int
+        // interaction issues today.  Deserves more of a look...
+        // Does our standalone parallel iterator not have the same
+        // type flexibility as the serial iterator?
+        //
+        for i in d[(...dom.ranges)] do
```

Comment:
For me though, even switching to a zippered forall, ra still fails.

Compiling with `--print-callstack-on-error` it looks like we're getting errors in the standalone iterator from this [line] (https://github.com/chapel-lang/chapel/blob/master/modules/internal/DefaultRectangular.chpl#L232) and an indentical [line] (https://github.com/chapel-lang/chapel/blob/master/modules/internal/DefaultRectangular.chpl#L425) in the follower. In the standalone case, changing this [line] (https://github.com/chapel-lang/chapel/blob/master/modules/internal/DefaultRectangular.chpl#L230) to be `idxType` instead of `strType` seems to resolve the error. That was just me randomly wondering if that was the issue, I didn't really look through the code and think about if that was the right decision.

But anyways, yeah I agree this can all be done in a seperate commit, and that'd it be nice to just get this fix in